Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM golang:1.19.3-alpine3.16 as builder
+FROM golang:1.19.4-alpine3.16 as builder
```

Comment:
@mzedel It works now :partying_face: 

(ref https://github.com/mendersoftware/integration-test-runner/pull/201#issuecomment-1369994140 )